### DISCUSSION TITLE
Documenting Data and Model Versions

### DISCUSSION THREAD

**Discussion by jds485** [USGS-R/drb-inland-salinity-ml#7] [General]
Code version control is tracked by git.
Data version control is tracked within S3
We need to document which data versions are used with which model versions.

I'm thinking of tracking the following in a GitHub-shared spreadsheet that we fill out for each model run:
Date Model Run,	Model Run Name,	Description,	Git Commit Date,	Git Commit ID,	`target` Name for Model, 	Path to a copy of `_targets` folder after model run (R-readable inputs and outputs)

Can also include S3 date pulled and any other info needed to know which data versions were used (I'm not yet sure what's needed).

Is this a good idea? Any other variables we should track?

**Comment by lekoenig** [comment]
I like the idea of adding the S3 date pulled or commit date of the input data versions. It'd be great to add this discussion to the agenda for one of our weekly meetings as we get a better sense of the data version info we'll need.
